maths/linear_diophantine_check.py

```diff
@@ -0,0 +1,101 @@
+"""
+   Program to check for solutions
+   of linear diophantine equations
+   can be possible or not
+"""
+
+# importing inbuilt gcd function
+from math import gcd
+
+"""
+   This function checks if integral
+   solutions are possible
+   >>>a,b,c, are parameters represents
+   >>>represents the variables in equation
+   ax + by = c
+"""
+
+
+def is_possible(a, b, c) -> bool:
+    """
+    returns true if solutions exists otherwise false
+    param a,b,c if first,second,third coeff. of equation
```

Comment:
Put the test cases in this docstring as doctests rather than writing separate functions for each of them.

Also, add more test cases than just 3